static/js/apps/eval_rig/feedback_form.tsx

```diff
@@ -35,7 +34,7 @@ export interface EvalInfo {
   dcStat: string;
 }
 
-interface Response extends EvalInfo {
+export interface Response extends EvalInfo {
```

Comment:
curious why exporting this interface because I don't see Response used in any other files?

Reply:
it's used in data_store for typedef